packages/material-ui/src/Chip/Chip.js

```diff
@@ -151,6 +152,8 @@ export const styles = (theme) => {
         marginRight: 3,
       },
     },
+    /* Styles applied to the root element if `variant="default"`. */
+    default: {},
```

Comment:
`filled` is probably a better working for this.. @oliviertassinari @mbrookes what do you think? If you agree I can prepare a PR afterwards for this.

Reply:
Sounds good.